test/framework/options.py

```diff
@@ -2271,7 +2271,7 @@ def test_include_module_naming_schemes(self):
             '--avail-module-naming-schemes',
         ]
 
-        env_cmd = "export PATH='%s'; export PYTHONPATH='%s'; " % (path, pythonpath)
+        env_cmd = "export PATH='%s' export PYTHONPATH='%s'; " % (path, pythonpath)
```

Comment:
No, env_cmd = "export PATH='%s' PYTHONPATH='%s'; " % (path, pythonpath)

Reply:
needs more coffee...

Reply:
fixed in 776993e6e with force push